README.md

```diff
@@ -42,6 +42,27 @@ Several example notebooks are provided in the examples/ directory. Opening these
 
 Several examples can be found in the `examples/` directory (the [Fitting impedance spectra notebook](https://impedancepy.readthedocs.io/en/latest/examples/fitting_example.html) is a great place to start) and the documentation can be found at [impedancepy.readthedocs.io](https://impedancepy.readthedocs.io/en/latest/).
 
+## Citing impedance.py
+
+[![DOI](https://joss.theoj.org/papers/10.21105/joss.02349/status.svg)](https://doi.org/10.21105/joss.02349)
+
+If you use impedance.py in published work, please consider citing https://joss.theoj.org/papers/10.21105/joss.02349 as
+
+```bash
+@article{Murbach2020,
```

Comment:
Is this supposed to render as a journal citation (ex: Murbach, M.D., et al., "impedance.py: A Python package for electrochemical impedance analysis". J. Open Source Soft., ...)?

Reply:
It's actually just the raw .bibtex that you could use directly (took inspiration from this citing section from Altair [here](https://github.com/altair-viz/altair#citing-altair) 😛)